namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "formbuilder_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.0"
+  repo_name = "formBuilder-product-page"
+  team_name = "formbuilder"
+}
+
+resource "kubernetes_secret" "formbuilder_ecr_credentials" {
+  metadata {
+    name      = "formbuilder_ecr_credentials_output"
```

Comment:
Unfortunately, Terraform doesn't like the _ in your secret name. Can you please change?
```
kubernetes_secret.formbuilder_ecr_credentials: metadata.0.name a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Reply:
Changed the metadata name